Assets/MixedRealityToolkit-SDK/Features/Audio/Influencers/AudioLoFiSourceQualityType.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Audio.Influencers
```

Comment:
We should probably move this into core as well.

Reply:
no. this is SDK only. it is 100% addon